AutoscoperM/AutoscoperM.py

```diff
@@ -899,11 +912,20 @@ def progressCallback(x):
             segmentVolume.SetName(segmentName)
             filename = os.path.join(outputDir, volumeSubDir, segmentName + ".tif")
             IO.castVolumeForTIFF(segmentVolume)
+
+            # Remove spacing before writing volume out
```

Comment:
Can the comment be improved to explain the rational for only removing spacing ? 

Should we instead use code like the following:

```python
    identityMatrix = vtk.vtkMatrix3x3()
    segmentVolume.SetDirectionMatrix(identityMatrix)
    segmentVolume.SetOrigin((0, 0, 0))
    segmentVolume.SetSpacing((1.0, 1.0, 1.0))
```